lutris/util/dxvk.py

```diff
@@ -1,23 +1,41 @@
 """DXVK helper module"""
 import os
+import json
 import time
 import shutil
+import urllib.request
 
-from requests import get
 from lutris.settings import RUNTIME_DIR
 from lutris.util.log import logger
 from lutris.util.extract import extract_archive
 from lutris.util.downloader import Downloader
 
-# Get latest DXVK versions from GitHub
-dxvk_url = "https://api.github.com/repos/doitsujin/dxvk/tags"
-dxvk_json = get(dxvk_url).json()
-dxvk_versions = [x['name'].replace('v', '') for x in dxvk_json]
 
-DXVK_LATEST = dxvk_versions[0]
-DXVK_PAST_RELEASES = [x['name'].replace('v', '') for x in dxvk_json]
+def get_latest_dxvk_versions():
+    """Get latest DXVK versions from GitHub"""
+    dxvk_url = "https://api.github.com/repos/doitsujin/dxvk/tags"
+    cache = os.path.join(RUNTIME_DIR, 'dxvk', 'dxvk_versions.json')
 
+    # If the DXVK cache does not exist then download it
+    if not os.path.exists("./dxvk_versions.json"):
```

Comment:
Why are you not checking the full file path?  Substitue "./dxvk_versions.json" with cache

Reply:
Good idea!